### PR TITLE
Update versioning for PostgreSQL, Leap and Salt and Migration info

### PR DESCRIPTION
**Pull request by jcayouette**
# Description

Provides updates for: (https://github.com/SUSE/spacewalk/issues/21563)

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/installation-and-upgrade/pages/db-migration-xy.adoc [hunk 1]
```diff
@@ -4,11 +4,14 @@
 This section covers upgrading the PostgreSQL database to the latest version.
 If you are already using PostgreSQL 14, you do not need to perform this migration.
 
-If you want to upgrade to the latest {productname} version, you must be using PostgreSQL version 13 or 14, depending on the underlying operating system:
+If you want to upgrade to the latest {productname} version, you must be using PostgreSQL version 13, 14, or 15 depending on the underlying operating system:
 
 * If you are running SLES 15 SP4, use PostgreSQL 14.
```
Comment (deneb-alpha) [inline modules/installation-and-upgrade/pages/db-migration-xy.adoc:9]:
```suggestion
* If you are running SUSE Linux Enterprise Server 15 SP4, use PostgreSQL 14.
```

File: modules/installation-and-upgrade/pages/db-migration-xy.adoc [hunk 2]
```diff
@@ -4,11 +4,14 @@
...
 * If you are running {opensuse} Leap 15.4, use PostgreSQL 14.
 
+* If you are running SLES 15 SP5, use PostgreSQL 15.
```
Comment (deneb-alpha) [inline modules/installation-and-upgrade/pages/db-migration-xy.adoc:12]:
```suggestion
* If you are running SUSE Linux Enterprise Server 15 SP5, use PostgreSQL 15.
```

**Review by deneb-alpha — CHANGES_REQUESTED** [review]
please, see  the comments I left

File: modules/installation-and-upgrade/pages/db-migration-xy.adoc (hunk 2 above)
Comment (jcayouette) [reply modules/installation-and-upgrade/pages/db-migration-xy.adoc:12]: There is a reason we have attribute. :) Woops!

File: parameters.yml [hunk 3]
```diff
@@ -167,8 +167,12 @@ targets: #Other makefile target not strictly related to book. These targets will
   - name: pdf-all-uyuni
     pre_lang_prerequisite: configure-uyuni
 asciidoc:
+  - attribute: opensuse-version
+    value: 15.5
   - attribute: saltversion
     value: 3006
```
[the remark this answers is not in the document]
Comment (jcayouette) [reply parameters.yml:173]: Thanks, I will go ahead and update relevant branches with the new version.

File: parameters.yml [hunk 4]
```diff
@@ -167,10 +167,12 @@ targets: #Other makefile target not strictly related to book. These targets will
...
     pre_lang_prerequisite: configure-uyuni
 asciidoc:
-  - attribute: postgresql
-    value: 15
+  - attribute: opensuse-version
+    value: 15.5
   - attribute: saltversion
     value: 3006.0
+  - attribute: postgresql
+    value: 15
```
Comment (keichwa) [inline parameters.yml:175]:
```suggestion
  - attribute: postgresql
    value: PostgreSQL
  - attribute: postgresql-version
    value: 15
```

**Review by keichwa — APPROVED** [review]

**Pull request merged**